src/alloc.rs

```diff
@@ -17,11 +17,9 @@ use crate::{
 
 pub extern crate alloc;
 
-// SAFETY: All zeros is equivalent to `None` (option layout optimization guarantee).
-//
-// In this case we are allowed to use `T: ?Sized`, since all zeros is the `None` variant and there
-// is no problem with a VTABLE pointer being null.
-unsafe impl<T: ?Sized> ZeroableOption for Box<T> {}
+// SAFETY: All zeros is equivalent to `None` (option layout optimization guarantee:
+// https://doc.rust-lang.org/stable/std/option/index.html#representation).
```

Comment:
```suggestion
// <https://doc.rust-lang.org/stable/std/option/index.html#representation>).
```
This helps kitty pick up the link correctly (otherwise the `)` is included in the url).

AFAIK this is also how one normally does links in markdown, right?

Reply:
Yeah, in that case we should also do the other one in `src/lib.rs` too, though it changes w.r.t. the kernel (where we have a few instances of links not within `<>` in normal comments) -- I can add a commit here or in another PR as you prefer.

Reply:
Done both -- https://github.com/Rust-for-Linux/linux/issues/1153. Though if you want to have the `Suggested-by` there, I am happy to.